app/src/main/kotlin/com/wire/android/ui/common/bottomsheet/conversation/ConversationSheetState.kt

```diff
@@ -74,6 +74,7 @@ fun rememberConversationSheetState(
                             conversationId = conversationId,
                             isFromTheSameTeam = isFromTheSameTeam,
                             isPrivate = conversationItem.isPrivate,
+                            isSelfUserTeamAdmin = false // TODO: Get actual value
```

Comment:
This is a super annoying value to get.

Most of the UI calling this function don't have a clue about user role within the team. So a compromise was made: to hardcode false.

The only implication is that a user that is a team admin won't be able to delete channels from the bottomsheet when long clicking on a Conversation, in the Conversation List.

In the future, as said before, we should refactor this.
We should have less generic composables/elements, and bring the logic to a higher layer, so instead of passing stuff like `isSelfUserTeamAdmin` or `isFromTheSameTeam`, we should pass meaningful values, like `isDeletePossible` to the presentation layer.